### ISSUE TITLE
Scancodes are private and thus unusable

### ISSUE THREAD

**Issue by andreymal** [jeremyletang/rust-sfml#311]
I tried to make a typical event handling:

```rust
match event {
    sfml::window::Event::KeyPressed { scan, .. } => {
        match scan {
            sfml::ffi::window::Scancode::Escape => { /* ... */ },
            _ => { /* ... */ },
        }
    },
    _ => { /* ... */}
}
```

...but got an ```error[E0603]: module `ffi` is private```

I guess someone forgot to add `pub use crate::ffi::window::Scancode;` in `src/window/keyboard.rs`

(UPD: and then `pub use self::keyboard::Scancode` in `src/window/mod.rs`)

**Comment by andreymal** [comment]
Oh no, `scan` is not even implemented for `KeyReleased`

**Comment by crumblingstatue** [comment]
This should be resolved by #313. Let me know if you have any problems!
